### ISSUE TITLE
508-defect-2 [TTY]: GI Bill Statement of Benefits - TTY: 711 link for deaf/hard-of-hearing users

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#79972]
## Background
We’re missing TTY info with the phone number at the bottom (last red highlight in screenshot). This is an accessibility problem.

<img width="433" alt="post-911-gibill" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/55992116/e9283e1e-21e3-4b24-93ef-ee02b2c71d7b">

## AC

- [ ] TTY is added to the page following the [guidance from the design system](https://design.va.gov/components/telephone#formatting)
- [ ] Reviewed and approved by CAIA
